nmostesting/suites/IS0502Test.py

```diff
@@ -593,3 +594,206 @@ def test_13(self, test):
+
+        flow_map = {flow["id"]: flow for flow in self.is04_resources["flows"]}
+        source_map = {source["id"]: source for source in self.is04_resources["sources"]}
+
+        try:
+            for resource in self.is04_resources["senders"]:
+                if not resource["transport"].startswith("urn:x-nmos:transport:rtp"):
+                    continue
+                if resource["flow_id"] is None:
+                    continue
+
+                flow = flow_map[resource["flow_id"]]
+                source = source_map[flow["source_id"]]
+
+                is05_transport_file = None
+                url_path = self.connection_url + "single/senders/" + resource["id"] + "/transportfile"
+                valid, result = self.do_request("GET", url_path)
+                if valid and result.status_code != 404:
+                    is05_transport_file = result.text
+                else:
+                    return test.FAIL("Unable to download transport_file for Sender ID {}".format(resource["id"]))
+
+                payload_type = None
+                for sdp_line in is05_transport_file.split("\n"):
+                    sdp_line = sdp_line.replace("\r", "")
+                    try:
+                        payload_type = int(re.search(r"^a=rtpmap:(\d+) ", sdp_line).group(1))
+                    except Exception:
+                        pass
+                if not payload_type:
+                    return test.FAIL("Unable to locate payload type from rtpmap in SDP file")
+
+                for sdp_line in is05_transport_file.split("\n"):
+                    sdp_line = sdp_line.replace("\r", "")
+                    if sdp_line.startswith(r"a=rtpmap:"):
+                        # Perform a coarse check first
+                        rtpmap = re.search(r"^a=rtpmap:\d+ {}/.+$".format(flow["media_type"].split("/")[1]),
+                                           sdp_line)
+                        if not rtpmap:
+                            return test.FAIL(r"a=rtpmap does not match Flow media type {} for Sender {}"
+                                             .format(flow["media_type"], resource["id"]))
+
+                        if source["format"] == "urn:x-nmos:format:video":
+                            rtpmap = re.search(r"^a=rtpmap:\d+ {}/90000$".format(flow["media_type"].split("/")[1]),
+                                               sdp_line)
+                            if not rtpmap:
+                                return test.FAIL("a=rtpmap clock rate does not match expected rate for Flow media "
+                                                 "type {} and Sender {}".format(flow["media_type"], resource["id"]))
+                        elif source["format"] == "urn:x-nmos:format:audio":
+                            if re.search(r"^audio\/L\d+$", flow["media_type"]):
+                                rtpmap = re.search(r"^a=rtpmap:\d+ L(\d+)\/(\d+)\/(\d+)$", sdp_line)
```

Comment:
Strictly speaking I think the second slash and `encoding parameters` are optional for audio, as per [RFC 4566 Section 6](https://tools.ietf.org/html/rfc4566#section-6):
```
         For audio streams, <encoding parameters> indicates the number
         of audio channels.  This parameter is OPTIONAL and may be
         omitted if the number of channels is one, provided that no
         additional parameters are needed.
```

Reply:
So they are, didn't know that.